net/net.go

```diff
@@ -165,11 +166,16 @@ func AddIPOffset(base *big.Int, offset int) net.IP {
 }
 
 // RangeSize returns the size of a range in valid addresses.
+// returns the size of the subnet or max value of int64
```

Comment:
need to clarify the circumstances in which MaxInt64 is returned, e.g.:
```suggestion
// returns the size of the subnet (or math.MaxInt64 if the range size would overflow int64)
```